Controller/RecipesController.php

```diff
@@ -95,7 +95,7 @@ public function view($id = null, $servings=null) {
                             'fields' => array('name')
                         ),
                         'Unit' => array(
-                            'fields' => array('name')
+                            'fields' => array('name', 'abbreviation') //2017-01-23 - added 'abbreviation' to be able to view those in the recipe view.
```

Comment:
Remove the comments.  We have source control for comments.  I know you mean well, but if every change was dated in code it quickly becomes unreadable overall.  Same goes for the other comments/dates.